packages/svelte/src/compiler/phases/3-transform/client/visitors/RegularElement.js

```diff
@@ -607,6 +613,13 @@ function build_element_attribute_update_assignment(element, node_id, attribute,
 		);
 	}
 
+	const inlinable_expression =
+		attribute.value === true
```

Comment:
shouldn't the comparison with true be part of the is_inlinable_expression function? also confused how it can be both a boolean `true` or an AST.ExpressionTag

Reply:
Attributes with `value ==== true` are the expressionless ones eg

```svelte
<div draggable />
```

Reply:
and how would that be different from `draggable={true}` or `draggable={false}` or `draggable="foo"` ? my main question is why it is not inside the function though but tested here outside of it.